pkg/cmd/codespace/ports.go

```diff
@@ -428,3 +429,21 @@ func normalizeJSON(j []byte) []byte {
 	// remove trailing commas
 	return bytes.ReplaceAll(j, []byte("},}"), []byte("}}"))
 }
+
+// startPortForwardingTCPListener starts a localhost tcp listener and returns the listener and bound port
+// Note: this is a re-implementation of `codespaces.ListenTCP` except that it allows all interfaces instead of just 127.0.0.1
+func startPortForwardingTCPListener(port int) (*net.TCPListener, int, error) {
```

Comment:
Instead of duplicating this method, consider adding a parameter to the `codespaces.ListenTCP` implementation to listen on `127.0.0.1` or on all interfaces.